geoportal/c2cgeoportal_geoportal/scaffolds/create/tilegeneration/config.yaml.tmpl_tmpl

```diff
@@ -97,7 +97,15 @@ generation:
     default_cache: s3
 
 redis:
-    url: 'redis://${REDIS_HOST}:${REDIS_PORT}/${REDIS_DB}'
+    socket_timeout: ${REDIS_TIMEOUT}
+    # For standalone version
+    # url: 'redis://${REDIS_HOST}:${REDIS_PORT}/${REDIS_DB}'
+    # For OpenShift version
+    sentinels:
+      - - ${REDIS_HOST}
+        - ${REDIS_PORT}
```

Comment:
Same syntax `- -` but with `-` on second line.

Reply:
In JSON it is: `[["host", "port"]]` :-)